.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
   release:
     types: [created]
 
+env:
+  FORMULA: cbmc-viewer
+  TAP: aws/tap
+  BOT_USER: aws-viewer-for-cbmc-release-ci
+  RELEASE_TAG: ${GITHUB_REF/refs\/tags\/}
+  VERSION: $(echo ${{ github.ref }} | cut -d "/" -f 3 | cut -d "-" -f 2)
+  FORK_REPO: https://${{ secrets.RELEASE_CI_ACCESS_TOKEN }}@github.com/$BOT_USER/homebrew-$(echo $TAP |cut -d "/" -f 2).git
+
 jobs:
   homebrew-pr:
     name: Homebrew Bump Formula PR
     runs-on: macos-10.15
     steps:
-      - name: Get release tag name
-        run: echo "RELEASE_TAG=${GITHUB_REF/refs\/tags\/}" >> $GITHUB_ENV
       - name: Configure git user name and email
         uses: Homebrew/actions/git-user-config@master
         with:
-          username: aws-viewer-for-cbmc-release-ci
+          username: ${{ env.BOT_USER }}
+
       - name: Create homebrew PR
         run: |
+          brew tap ${{ env.TAP }}
           brew update-reset
-          brew tap aws/tap
-          brew bump-formula-pr --tag "$RELEASE_TAG" --revision "$GITHUB_SHA" aws/tap/cbmc-viewer
+          brew bump-formula-pr --tag "${{ env.RELEASE_TAG }}" --revision "$GITHUB_SHA" ${{ env.TAP }}/${{ env.FORMULA }}
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.RELEASE_CI_ACCESS_TOKEN }}
+
+  build-bottle:
+    needs: homebrew-pr
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-10.15]
+    runs-on: ${{ matrix.os }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Checkout PR
+        run: |
+          brew tap ${{ env.TAP }}
+          brew update-reset
+          cd $(brew --repo ${{ env.TAP }})
+          git remote add fork-repo ${{ env.FORK_REPO }}
+          git fetch fork-repo
+          git checkout -B bump-${{ env.FORMULA }}-${{ env.VERSION }} fork-repo/bump-${{ env.FORMULA }}-${{ env.VERSION }}
+
+      - name: Tap Syntax
+        run: |
+          brew audit --online --git --skip-style ${{ env.TAP }}/${{ env.FORMULA }}
+          brew style ${{ env.TAP }}/${{ env.FORMULA }}
+
+      - name: Build bottle
+        run: |
+          brew test-bot --tap ${{ env.TAP }} --testing-formulae ${{ env.TAP }}/${{ env.FORMULA }} --only-formulae --root-url=https://github.com/$GITHUB_REPOSITORY/releases/download/${{ env.RELEASE_TAG }}
```

Comment:
This URL should probably be one of your environment variable definitions along with the others at the top of the file.

Reply:
- This URL is completely generated by the other information generated by other information provided.
- If i put `ROOT_URL` in `env`, i won't be able to replace `${{ env.RELEASE_TAG }}` as `$RELEASE_TAG` because of the reason i mentioned above.
> The above code will not work as `$RELEASE_TAG` will be replaced by `${GITHUB_REF/refs\/tags\/}` as string.
> 
> ```
> $(echo `${GITHUB_REF/refs\/tags\/}` | cut -d "-" -f 2) = '${GITHUB_REF/refs\/tags\/}'
> ```